### DISCUSSION TITLE
deepin-unioncode希望能提供Windows平台的版本

### DISCUSSION THREAD

**Discussion by swt1me** [linuxdeepin/developer-center#8564] [Features Request & Ideas | 特性请求 & 头脑风暴]
主要是Windows平台好用的IDE太少了JetBrains IDE开发其他语言还要换IDE VScode项目大了体验真的很烂

**Comment by BLumia** [comment]
- https://github.com/linuxdeepin/developer-center/discussions/5430
- https://github.com/linuxdeepin/developer-center/discussions/8365
- https://github.com/linuxdeepin/developer-center/discussions/6630
- https://github.com/linuxdeepin/developer-center/discussions/5445
